### PR TITLE
Fix some issues

### PR DESCRIPTION
**Pull request by Ariadne330**
According to  suggestions in this [issue](https://github.com/intel-analytics/bigdl-llm-tutorial/issues/55), I made some changes in related files. See in details in the above page.

### CODE REVIEW

File: ch_5_AppDev_Intermediate/5_1_ChatBot.ipynb [hunk 1]
```diff
@@ -105,10 +105,11 @@
     "\n",
     "> **Note**\n",
     ">\n",
-    "> * Currently, `load_in_low_bit` supports options `'sym_int4'`, `'asym_int4'`, `'sym_int5'`, `'asym_int5'` or `'sym_int8'`, in which 'sym' and 'asym' differentiate between symmetric and asymmetric quantization. Option `'nf4'` is also supported, referring to 4-bit NormalFloat.\n",
+    "> * Currently, `load_in_low_bit` supports options `'sym_int4'`, `'asym_int4'`, `'sym_int5'`, `'asym_int5'` or `'sym_int8'`, in which 'sym' and 'asym' differentiate between symmetric and asymmetric quantization. `'nf3'` and `'nf4'` stand for normalFloat quantizations . Floating point precision `'fp4'`, `'fp8'`, `'fp16'` and Option `'mixed_4bit'` are also supported.\n",
```
Comment (hkvision) [inline ch_5_AppDev_Intermediate/5_1_ChatBot.ipynb:108]: may exclude nf3 for the time being. i.e. Keep `Option `'nf4'` is also supported, referring to 4-bit NormalFloat`.
Floating point precision -> floating point precision__s__
and mixed precisions including mixed_fp4 and mixed_fp8 are ...

File: ch_6_GPU_Acceleration/6_1_GPU_Llama2-7B.md [hunk 2]
```diff
@@ -17,6 +17,13 @@ pip install bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
 > **Note**
 > The above command will install `intel_extension_for_pytorch==2.0.110+xpu` as default
 
+It is also necessary to set OneAPI environment variables for BigDL-LLM on Intel GPUs.
```
Comment (hkvision) [inline ch_6_GPU_Acceleration/6_1_GPU_Llama2-7B.md:20]: OneAPI -> oneAPI
necessary -> required

File: ch_6_GPU_Acceleration/6_1_GPU_Llama2-7B.md [hunk 3]
```diff
@@ -17,6 +17,13 @@ pip install bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
...
+
+```bash
+# configure OneAPI environment variables
```
Comment (hkvision) [inline ch_6_GPU_Acceleration/6_1_GPU_Llama2-7B.md:23]: One -> one, same for other places

File: ch_6_GPU_Acceleration/6_1_GPU_Llama2-7B.md [hunk 4]
```diff
@@ -78,7 +85,7 @@ model_in_8bit_gpu = model_in_8bit.to('xpu')
 ```
 
 > **Note**
-> * Currently, BigDL-LLM on Intel GPUs has supports options `'sym_int4'`, `'asym_int4'`, `'sym_int5'`, `'asym_int5'` or `'sym_int8'`, in which 'sym' and 'asym' differentiate between symmetric and asymmetric quantization. Option `'nf4'` is also supported, referring to 4-bit NormalFloat.
+> * Currently, `load_in_low_bit` supports options `'sym_int4'`, `'asym_int4'`, `'sym_int5'`, `'asym_int5'` or `'sym_int8'`, in which 'sym' and 'asym' differentiate between symmetric and asymmetric quantization. `'nf3'` and `'nf4'` stand for normalFloat quantizations . Floating point precision `'fp4'`, `'fp8'`, `'fp16'`, `'mixed_fp4'` and `'mixed_fp8'` are also supported.
```
Comment (hkvision) [inline ch_6_GPU_Acceleration/6_1_GPU_Llama2-7B.md:88]: same as above

**Comment by Oscilloscope98** [comment]
Please remove the preview block of audio in the speech recognition section

File: ch_5_AppDev_Intermediate/5_2_Speech_Recognition.ipynb [hunk 5]
```diff
@@ -142,18 +130,18 @@
     "\n",
     "Once you have optimized the Whisper model using BigDL-LLM with INT4 optimization and loaded the Whisper processor, you are ready to begin transcribing the audio through model inference.\n",
     "\n",
-    "Let's start with the English audio file `audio_en.mp3`. Before we feed it into Whisper processor, we need to extract sequence data from raw speech waveform:"
+    "Let's start with the English audio file `audio_en.wav`. Before we feed it into Whisper processor, we need to extract sequence data from raw speech waveform:"
    ]
```
Comment (plusbang) [inline ch_5_AppDev_Intermediate/5_2_Speech_Recognition.ipynb:134]: I feel it maybe better to provide such `wget -O audio_en.mp3 https://datasets-server.huggingface.co/assets/common_voice/--/en/train/5/audio/audio.mp3` command to make how to get `audio_en.mp3` more clear.

Comment (Ariadne330) [reply ch_5_AppDev_Intermediate/5_2_Speech_Recognition.ipynb:134]:
```shell
wget --no-check-certificate -O audio.wav https://datasets-server.huggingface.co/assets/carlot/AIShell/--/692ef58020d79b21f54eb25b15a4813d4f9650d7/--/default/train/84/audio/audio.wav
--2023-12-15 17:07:07--  https://datasets-server.huggingface.co/assets/carlot/AIShell/--/692ef58020d79b21f54eb25b15a4813d4f9650d7/--/default/train/84/audio/audio.wav
正在解析主机 child-prc.intel.com (child-prc.intel.com)... 10.239.120.56
正在连接 child-prc.intel.com (child-prc.intel.com)|10.239.120.56|:913... 已连接。
OpenSSL: error:0A000126:SSL routines::unexpected eof while reading
无法建立 SSL 连接。
```
When using `wget` command It is more likely to get network problem so I provided the link to download directly -_-， maybe I can add a note 'click the link to download the examples' or just leave it alone?

**Pull request merged**